modules/local/virusbreakend/main.nf

```diff
@@ -0,0 +1,55 @@
+// NOTE(SW): the --db argument for the virusbreakend command must have a trailing slash if it is a symlink
+
+process VIRUSBREAKEND {
+    tag "${meta.id}"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "nf-core/gridss:2.13.2--1"
```

Comment:
Why is this using an nf-core docker container (and therefore no singularity)

Reply:
The Bioconda CI Azure instances used to build the packages and Docker/Singularity images have insufficient disk space to build this image. As an alternative I arranged with Maxime to have the image I built myself uploaded to the nf-core Quay.io account.

<br />

- For more information see: https://github.com/nf-core/oncoanalyser/pull/12

Reply:
Mmm.
Can we put the corresponding singularity image somewhere @maxulysse ?
I've just had a failure because this is a 2Gb image, and it takes too long to pull and convert docker -> singularity.

Reply:
If you wanted to do some testing/confirmation, the original Singularity image I had in that PR is still available from `https://pub-29f2e5b2b7384811bdbbcba44f8b5083.r2.dev/singularity/gridss:2.13.2--1`

Reply:
Marking this as resolved for now, please reopen if you'd like to continue the discussion here.